app/concepts/cinema_halls/use_cases/generate_seats.rb

```diff
@@ -0,0 +1,32 @@
+module CinemaHalls
+  module UseCases
+    class GenerateSeats
+      attr_reader :params
+
+      def initialize(params:)
+        @params = params
+      end
+
+      def call
+        columns = to_range(params[:columns])
```

Comment:
I'm wondering why do you need to generate a range when your columns and rows are already arrays - you could just map them directly 🤔